install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ $(whoami) != "root" ]]; then
+  echo "This file needs to be executed as root or with sudo."
+  exit
+fi
+
+defaultdir="/usr/bin"
+
+echo "Directory to install to (default: $defaultdir)"
+read installdir
+
+if [[ $installdir = "" ]]; then
+  installdir=$defaultdir
+fi
+
+if [[ ! -d $installdir ]]; then
+  echo "This directory does not exist."
+  exit
+fi
+
+cd src
+zip -r ../pridefetch.zip *
+cd ..
+echo "#!/usr/bin/env python" | cat - pridefetch.zip > pridefetch
+cp pridefetch "$installdir/"
+chmod 755 "$installdir/pridefetch"
+
+if [[ -f "$installdir/pridefetch" ]]; then
+  echo -e "\nPridefetch installed successfully"
+  echo -e "\n\nMake sure $installdir is in PATH\nTo add directory to path, add this to your shells rc file:"
+  echo -e "\nexport PATH=\"$installdir:\$PATH\""
```

Comment:
This isn't always needed, and almost certainly won't be needed with the default install directory of `/usr/bin`; could we check if the install dir is already in path before telling the user to add it to their shell's .rc file?